src/utils/requests/unwaterTree.ts

```diff
@@ -0,0 +1,33 @@
+import { createAPIUrl } from '../createAPIUrl';
+import { requests } from '../requestUtil';
+
+type UnwaterTreeSignature = (params: {
+  id: string;
+  token: string;
+  userId: string;
+  wateringId: number;
+}) => Promise<void>;
+
+export const unwaterTree: UnwaterTreeSignature = async ({
+  id,
+  token,
+  userId,
+  wateringId,
+}) => {
+  const urlUnadopt = createAPIUrl(
```

Comment:
Why this naming? `urlUnadopt`? Looks like an copy paste error?

Reply:
True! Thx for the hint!